docs/workflow/README.md

```diff
@@ -1,6 +1,7 @@
 # Authoring Workflows with the Go SDK
 
 ## A simple two-step workflow
+//BROKEN LINK
```

Comment:
There are many broken links like this one

https://github.com/conductor-sdk/java-sdk-examples/blob/main/src/main/java/io/orkes/samples/quickstart/WorkflowManagement.java#L83-L92

Where should this example go? in this repo or `java-sdk-examples`?

(Will delete this line once we decide what to do with these broken link examples)